crutest/src/main.rs

```diff
@@ -1968,6 +2046,353 @@ async fn replay_workload(
     Ok(())
 }
 
+// Test that a downstairs can be replaced while the initial reconciliation
+// is underway.
+//
+// This test makes use of the dsc client to stop and start downstairs that
+// will allow us to create a mismatch between downstairs which will then
+// trigger a reconcile.
+async fn replace_while_reconcile(
+    guest: &Arc<Guest>,
+    wtq: &mut WhenToQuit,
```

Comment:
Could this be passed by value?  It looks like the caller doesn't use it afterwards.